### ISSUE TITLE
Unable to run project 

### ISSUE THREAD

**Issue by raju-nichit** [naturalprogrammer/lemon-demo-angular1#2]
Loading "grunt-karma.js" tasks...ERROR
>> TypeError: Cannot read property 'prototype' of undefined

Running "serve" task
Loading "grunt-karma.js" tasks...ERROR
>> TypeError: Cannot read property 'prototype' of undefined

**Comment by naturalprogrammer** [comment]
Yes, I too see that error on my machine, but the application runs ok.
